web/packages/design/src/Onboard/OnboardFooter.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import styled, { useTheme } from 'styled-components';
+
+import { Flex, Link, Text } from 'design';
+
+export const OnboardFooter = () => {
+  const theme = useTheme();
+  return (
+    <StyledFooter>
+      <StyledContent>
+        <Text typography={theme.typography.paragraph2}>
+          &copy; Gravitational, Inc. All Rights Reserved
+        </Text>
+        <StyledLink href={'https://goteleport.com/legal/tos/'} target="_blank">
+          Terms of Service
+        </StyledLink>
+        <StyledLink
+          href={'https://goteleport.com/legal/privacy/'}
+          target="_blank"
+        >
+          Privacy Policy
+        </StyledLink>
+      </StyledContent>
+    </StyledFooter>
+  );
+};
+
+const StyledContent = styled(Flex)`
+  justify-content: center;
+  width: 100%;
+  gap: 50px;
```

Comment:
Nit for the future: FWIW, if the theme doesn't have the exact spacing value, I usually try to round it to the closest available one or use multiples.

https://github.com/gravitational/teleport/blob/b1982ea2c0e25ef24933ab7011c78e48697481d9/web/packages/teleterm/src/ui/TopBar/TopBar.tsx#L59

Reply:
that's a good suggestion, i'll also do this 👍